### DISCUSSION TITLE
Set a margin-bottem css rule use missing.css?

### DISCUSSION THREAD

**Discussion by zw963** [bigskysoftware/missing#66] [Q&A] [answered]
Following screenshot is my sample project for start use missing.css.

![image](https://github.com/user-attachments/assets/5efff799-b879-4279-a036-c4e0fa283faf)

As you can see, the gap between the navbar and sidebar is very large. firefox told me, there are a very large margin-bottom exists, as show by following screenshot.

![image](https://github.com/user-attachments/assets/00d20a8f-883f-4fa0-9b2b-5986bd3d40d3)

So, i want to set `margin-bottom: 5px;`, like following:

```html
<header class="navbar" style="margin-bottom: 5px;"> 
     .....
</header>
```

get the result like following screenshot.

![image](https://github.com/user-attachments/assets/10d26286-9d76-4953-964b-38063756c02c)

how to use missing.css utility class achieve this instead of add a style attribute?

Thanks.

**Comment by Mohamedhazeem** [comment]
To achieve the same effect using a utility class from missing.css instead of an inline style attribute, you would first need to check if missing.css has a predefined utility class for margin settings (similar to what you might find in popular CSS frameworks like Tailwind CSS or Bootstrap).
For example, if missing.css provides margin utility classes like mb-5, you could apply that class directly to your <header> tag instead of using inline styles.
```
<header class="navbar mb-5">
    ...
</header>
```
This would set the margin-bottom to 5px if the class mb-5 is defined in missing.css for that value. Make sure to consult the documentation or CSS file of missing.css for the correct class name and value mapping.

By the way, if you're looking for an easier way to manage styling changes, you might want to check out TweakSync. It lets you visually tweak your webpage's appearance directly from Chrome in real-time, and then sync those changes back to VS Code with a click. It’s a real time-saver, especially for front-end developers who are tired of switching between tools. You can find it on both [Chrome Web Store](https://chromewebstore.google.com/detail/tweaksync/ponbgdgplifbgjianglonaondpkonnio?authuser=0&hl=en-GB) and [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=TweakSync.tweaksync). Feel free to give it a try.

**Reply by zw963** [reply]
>  you might want to check out TweakSync.

Thanks for let me know this tools!

> Make sure to consult the documentation or CSS file of missing.css for the correct class name and value mapping.

Yes, i did it before ask this question, because i could't found it after read through all doc, so i asked here.
so, there is no tailwind `mb-5` equivalent exists in missing.css ?

**Comment by geoffrey-eisenbarth** [comment, marked as answer]
You can adjust the margin (and padding) of an element by using one of the [density utilities](https://missing.style/docs/utils/#density). But honestly setting the `style` attribute is probably the correct answer.
